### DISCUSSION TITLE
POW-HVM2H-12V-N

### DISCUSSION THREAD

**Discussion by messageexplorer** [softwarecrash/Solar2MQTT-ESP8266#117] [General]
Hello, 
as I can see this solution supports POW-HVM1.5H-12, but I have POW-HVM2H-12V-N. Can someone advise is my inverter supported too?

Thanks in advance

**Comment by messageexplorer** [comment]
Seems my POW-HVM2H-12V-N supports PI30 protocol. Minimum QPIGS command works.

**Reply by softwarecrash** [reply]
when it works with PI30 commands, this software will work

**Reply by messageexplorer** [reply]
Thank you.
